adaptivefiltering/pdal.py

```diff
@@ -27,8 +27,9 @@ def execute_pdal_pipeline(dataset=None, config=None):
         The configuration of the PDAL pipeline, according to the PDAL documentation.
     :type config: dict
     :return:
-        A numpy array data structure containing the PDAL output.
-    :rtype: numpy.array
+        The full pdal pipeline object
+    :rtype: pipeline
```

Comment:
When the exact type is not known, I think it is better to omit the `:rtype:` statement.